python/torch_mlir_e2e_test/test_suite/slice_like.py

```diff
@@ -677,6 +677,78 @@ def SliceCopyNonZeroDim_Module_basic(module, tu: TestUtils):
 # ==============================================================================
 
 
+class SelectFloatCopyNonZeroDim_Module(torch.nn.Module):
```

Comment:
nit: the `NonZeroDim` part of the name in the `SliceCopy` tests above refers to the fact that the `dim` argument is non-zero. However, this test is testing `torch.ops.aten.select` with `dim=0`, so the name is not accurate.